src/rastervision/common/tasks/train_model.py

```diff
@@ -81,6 +81,13 @@ def get_lr(epoch):
             callback = LearningRateScheduler(get_lr)
             callbacks.append(callback)
 
+        if self.options.epoch_decay:
+            def get_lr(epoch):
+                decay_factor = 1 / (1.0 + self.options.epoch_decay * epoch)
+                return self.options.init_lr * decay_factor
+            callback = LearningRateScheduler(get_lr)
+            callbacks.append(callback)
```

Comment:
It seems like `lr_schedule`, `cyclic_lr`, `decay` and `epoch_decay` should be mutually exclusive options. Otherwise they will conflict with each other. You should raise an error in options if this happens.